src/Java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/processing/GregtechMetaTileEntity_IndustrialAlloySmelter.java

```diff
@@ -256,9 +256,6 @@ public boolean checkRecipeGeneric(ItemStack[] aItemInputs, FluidStack[] aFluidIn
 		// EU discount
 		float tRecipeEUt = (tRecipe.mEUt * aEUPercent) / 100.0f;
 		int tHeatCapacityDivTiers = mHeatingCapacity / 900;
-		if (tHeatCapacityDivTiers > 0) {
```

Comment:
Why remove heating coils bonus, is it broken?

Reply:
the tooltip said the coil only gives speed bonus, not eu bonus

Reply:
Isee